Integrations/integration-ServiceNow.yml

```diff
@@ -529,6 +528,33 @@ script:
         return dicRes;
     }
 
+
+    var SNGetGroups = function(name){
```

Comment:
make sure you use this argument to bring only group if argument is provided

Reply:
Fixed